spec/mailers/previews/placements/school_user_mailer_preview.rb

```diff
@@ -34,6 +50,25 @@ def user
     )
   end
 
+  def school_with_appetite(appetite, year)
```

Comment:
```suggestion
  def school_with_appetite(appetite, academic_year)
```
If we renamed this argument we'd be able to omit it from line 61